spec/models/ems_cluster_spec.rb

```diff
@@ -213,4 +213,30 @@
       expect(result).to be_falsey
     end
   end
+
+  context "#tenant_identity" do
+    # admin user is needed to process Events
```

Comment:
To clarify, I am **not** saying get rid of `admin`, I am saying the comment is no longer needed.